ui/src/common/index.js

```diff
@@ -27,3 +27,4 @@ export * from './useToggleStar'
 export * from './useTraceUpdate'
 export * from './Writable'
 export * from './SongSimpleList'
+export * from './artistSimpleList'
```

Comment:
Please rename the file to `ArtistSimpleList` (to keep consistency: PascalCase for classes, camelCase for functions).

Reply:
ok